### ISSUE TITLE
jsx-a11y plugin loading failed

### ISSUE THREAD

**Issue by alok1111** [jsx-eslint/eslint-plugin-jsx-a11y#878]
I'm facing a problem with loading `jsx-a11y` in eslint. The stack trace looks as follows:
```
/usr/bin/yarn run lint
yarn run v1.22.19
$ yarn eslint --ext .ts,.tsx app/frontend
$ /home/dev/IdeaProjects/some-project/web/node_modules/.bin/eslint --ext .ts,.tsx app/frontend

Oops! Something went wrong! :(

ESLint: 8.19.0

Error: Failed to load plugin 'jsx-a11y' declared in '.eslintrc » eslint-config-airbnb » /home/dev/IdeaProjects/some-project/web/node_modules/eslint-config-airbnb/rules/react-a11y.js': Cannot find module 'core-js-pure/stable/object/define-property'
Require stack:
- /home/dev/IdeaProjects/some-project/web/node_modules/@babel/runtime-corejs3/core-js-stable/object/define-property.js
- /home/dev/IdeaProjects/some-project/web/node_modules/aria-query/lib/index.js
- /home/dev/IdeaProjects/some-project/web/node_modules/eslint-plugin-jsx-a11y/lib/rules/aria-activedescendant-has-tabindex.js
- /home/dev/IdeaProjects/some-project/web/node_modules/eslint-plugin-jsx-a11y/lib/index.js
- /home/dev/IdeaProjects/some-project/web/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (/home/dev/IdeaProjects/some-project/web/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/home/dev/IdeaProjects/some-project/web/node_modules/@babel/runtime-corejs3/core-js-stable/object/define-property.js:1:80)
    at Module._compile (/home/dev/IdeaProjects/some-project/web/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Process finished with exit code 2
```
**package.json**
```
{
  "name": "some-project",
  "author": "author",
  "private": true,
  "scripts": {
    "dev": "webpack serve --mode development",
    "build": "webpack --mode production",
    "lint": "yarn eslint --ext .ts,.tsx app/frontend",
    "lint:fix": "yarn eslint --fix --ext .ts,.tsx app/frontend"
  },
  "dependencies": {
    "jquery": "2.2.4",
    "lodash": "4.17.21",
    "moment": "2.22.2",
    "moment-duration-format": "2.3.2",
    "react": "17.0.1",
    "react-dom": "17.0.1"
  },
  "devDependencies": {
    "@babel/core": "7.18.6",
    "@babel/preset-env": "7.18.6",
    "@babel/preset-react": "7.18.6",
    "@sentry/webpack-plugin": "1.18.2",
    "@types/jquery": "3.5.14",
    "@types/lodash": "4.14.182",
    "@types/react": "17.0.1",
    "@types/react-dom": "17.0.1",
    "@types/webpack-bundle-analyzer": "4.4.1",
    "@typescript-eslint/eslint-plugin": "5.30.6",
    "@typescript-eslint/parser": "5.30.6",
    "babel-loader": "8.2.5",
    "css-loader": "6.4.0",
    "eslint": "8.19.0",
    "eslint-config-airbnb": "19.0.4",
    "eslint-config-airbnb-typescript": "17.0.0",
    "eslint-plugin-import": "2.26.0",
    "eslint-plugin-import-newlines": "1.2.3",
    "eslint-plugin-jsx-a11y": "6.6.0",
    "eslint-plugin-react": "7.30.1",
    "eslint-plugin-react-hooks": "4.6.0",
    "eslint-plugin-simple-import-sort": "7.0.0",
    "eslint-webpack-plugin": "3.2.0",
    "html-webpack-plugin": "5.5.0",
    "less": "4.1.2",
    "less-loader": "10.2.0",
    "mini-css-extract-plugin": "2.4.2",
    "style-loader": "3.3.0",
    "styled-components": "3.4.10",
    "ts-loader": "9.3.1",
    "typescript": "4.7.4",
    "webpack": "5.73.0",
    "webpack-bundle-analyzer": "4.5.0",
    "webpack-cli": "4.10.0",
    "webpack-dev-server": "4.9.3",
    "webpack-merge": "5.8.0"
  }
}
```
Current workaround is to add `"core-js-pure": "3.23.4",` to `devDependencies`.
Am I understanding right that the issue comes from `aria-query` dependency?

**Comment by rapidfixer** [comment]
Double this, also have the same error

**Comment by ljharb** [comment]
@alok1111 are you still having this issue? if so, please make a repro repo, and i'll be happy to reopen.
